doc-Managing_Infrastructure_and_Inventory/topics/Data_Optimization_Features.adoc

```diff
@@ -0,0 +1,158 @@
+[[data-optimization-features]]
+= Data Optimization Features
+
+[[optimization]]
+== Optimization
+
+{product-title} optimization functions allow you to view
+utilization trends, and identify and project bottlenecks in your
+environment. In addition, you can predict where you have capacity for
+additional virtual machines.
+
```

Comment:
I recommend noting here that you need C&U enabled so that customers aren't confused (this is only because the content is now in a separate guide from the 'how to enable C&U' content.